packages/ckeditor5-autosave/src/autosave.js

```diff
@@ -224,12 +240,19 @@ export default class Autosave extends Plugin {
 	 * @private
 	 */
 	_save() {
+		if ( this._savePromise ) {
+			return this._savePromise;
```

Comment:
Let's make sure that the returned promise is resolved only after the data is truly saved (the one that was in the editor while the `save()` method was called.